kong.conf.default

```diff
@@ -1494,7 +1494,9 @@
 # By default, the DNS resolver will use the standard configuration files
 # `/etc/hosts` and `/etc/resolv.conf`. The settings in the latter file will be
 # overridden by the environment variables `LOCALDOMAIN` and `RES_OPTIONS` if
-# they have been set.
+# they have been set. To use these environment variables, set the option
+# `nginx_main_env = RES_OPTIONS; env LOCALDOMAIN`. Otherwise, Nginx will
+# removes these environment variables when starting worker processes.
```

Comment:
```suggestion
# they have been set. To use these environment variables, set the options
# `nginx_main_env = RES_OPTIONS;` and `env LOCALDOMAIN` in your nginx
# configuration template to prevent nginx from removing them during worker
# process start.
```
It seems to me that this suggestion, while workable, is pretty difficult.  Is there a good reason not to put these directives into the default nginx template?

Reply:
Originally I think this fix is more easy and stable for 3.8 release, that we have another DNS cname regression necessary to fix in next release. So I individually don't want to involve another code modification in DNS client.

Reply:
KK, I'll research how to introduce environment variables into the default template of Kong later.

Reply:
isn't all there is to do, adding:

	env RES_OPTIONS;
	env LOCALDOMAIN;

to the template? It doesn't even have to be configurable, right?

Reply:
That's it. Just like in your example, we just need to hard code them.